packages/core-components/src/layout/BottomLink/BottomLink.tsx

```diff
@@ -50,7 +50,13 @@ export type BottomLinkProps = {
   onClick?: (event: React.MouseEvent<HTMLAnchorElement>) => void;
 };
 
-/** @public */
+/**
+ * Footer with link used in  {@link InfoCard | InfoCard } and {@link TabbedCard | TabbedCard}
```

Comment:
All of these `X | X` can be shortened to just `X` (e.g. in this case, it could be just `{@link InfoCard}`

Reply:
Yes - they will map to this X() format with brackets , but it's not a big deal and code will be cleaner ;)